appsec/src/extension/ddtrace.c

```diff
@@ -248,14 +248,14 @@ static zval *_get_span_modifiable_array_property(
 {
 #if PHP_VERSION_ID >= 80000
     zval *res =
-        zobj->handlers->get_property_ptr_ptr(zobj, propname, BP_VAR_R, NULL);
+        zobj->handlers->get_property_ptr_ptr(zobj, propname, BP_VAR_IS, NULL);
```

Comment:
Looking at the code, when the span has been invalidated (which also isn't clear to me why that's happening), won't `get_property_ptr_ptr` return `NULL`? In which case this will crash instead.

Reply:
Ah no, it should be fine.